src/screens/Quiz/components/Lives.tsx

```diff
@@ -1,54 +1,103 @@
 import { useGameContext } from "hooks";
 import { useEffect, useRef } from "react";
-import gsap from "gsap";
+import { gsap, Cubic } from "gsap";
 import styles from "./Lives.module.scss";
 
 const { life, lifeLost } = styles;
 
 export const Lives = () => {
   const { remainingLives, totalLives } = useGameContext();
-  const livesContainerRef = useRef(null);
+  const droppingLivesContainerRef = useRef(null);
 
   useEffect(() => {
     const ctx = gsap.context(() => {
       if (remainingLives === 2) {
-        gsap.to(`[data-effect=life-2]`, { fill: lifeLost, duration: 0.3 });
+        gsap.to(`[data-effect=droppingLife-2]`, {
+          y: "100%",
+          duration: 0.3,
+          autoAlpha: 0,
+          rotate: 45,
+          ease: Cubic.easeIn,
+        });
       } else if (remainingLives === 1) {
-        gsap.set(`[data-effect=life-2]`, { fill: lifeLost });
-        gsap.to(`[data-effect=life-1]`, { fill: lifeLost, duration: 0.3 });
+        gsap.set(` [data-effect=droppingLife-2]`, {
+          autoAlpha: 0,
+        });
+
+        gsap.to(`[data-effect=droppingLife-1]`, {
+          y: "100%",
+          duration: 0.3,
+          autoAlpha: 0,
+          rotate: 45,
+          ease: Cubic.easeIn,
+        });
       } else if (remainingLives === 0) {
-        gsap.set(`[data-effect=life-2], [data-effect=life-1]`, {
-          fill: lifeLost,
+        gsap.set(`[data-effect=droppingLife-2], [data-effect=droppingLife-1]`, {
+          autoAlpha: 0,
+        });
+
+        gsap.to(`[data-effect=droppingLife-0]`, {
+          y: "100%",
+          duration: 0.3,
+          autoAlpha: 0,
+          rotate: 45,
+          ease: Cubic.easeIn,
         });
-        gsap.to(`[data-effect=life-0]`, { fill: lifeLost, duration: 0.3 });
       }
-    }, livesContainerRef);
+    }, droppingLivesContainerRef);
 
     return () => ctx.revert();
```

Comment:
use `ctx.kill()` in cleanup function instead, then remove `gsap.set(...)` commands